### DISCUSSION TITLE
Error tycho compiler plugins 2.7.2 eclipse java ERP

### DISCUSSION THREAD

**Discussion by miltonchirinos44** [dbeaver/dbeaver#16456] [General]
Hello community, I am trying to make some changes in the dbeaver but I have problems with the implementation of the dbeaver project in eclipse java ERP, I tell you that I am using:
-  openjdk 11
-  maven 3.6.3+
Find  guide in the following link https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse
I have a problem with step 10 when I import the dbeaver project as I get the following error
![Error- eclipse_workspace_Eclipse_IDE](https://user-images.githubusercontent.com/52106718/167864465-95101ab0-27d0-48b9-a373-c5bf7f3c64dd.png)

Has anyone else had the same error, thanks for your help.

**Comment by ShadelessFox** [comment]
Hello @miltonchirinos44,

Does changing the action to `Do Not Execute (add to workspace)` help?

**Reply by miltonchirinos44** [reply]
Hi @ShadelessFox , I don't understand what you're trying to tell me? I tell you that I was able to run the dbeaver project by adding these lines of code in the porm.xml
```
    <dependencies>
    	 <dependency>
	       <groupId>org.slf4j</groupId>
	       <artifactId>slf4j-api</artifactId>
	       <version>1.7.5</version>
	   </dependency>
	   <dependency>
	       <groupId>org.slf4j</groupId>
	       <artifactId>slf4j-log4j12</artifactId>
	       <version>1.7.5</version>
	   </dependency>
    </dependencies>
```

which allowed me to run the project, but I am aware that the error is present regarding this `tycho compiler` library and I need help to solve the problem that I mentioned at the beginning
![image](https://user-images.githubusercontent.com/52106718/168620332-ece43284-88fa-4aad-8a59-6138f1f40cb1.png)

**Reply by ShadelessFox** [reply]
Change `Resolve Later` to `Do Not Execute (add to workspace)`:

![image](https://user-images.githubusercontent.com/35821147/168621194-6107619e-3601-4ec8-ba22-3354f3e53b09.png)

**Reply by miltonchirinos44** [reply]
If I click on the option resolve later and that way it lets me import the project
![image](https://user-images.githubusercontent.com/52106718/168622498-4a4b164f-1caa-4573-b1bd-834a9706c127.png)

**Reply by miltonchirinos44** [reply]
El problema es cuando ejecuto `mvn clean install `me aparece el siguiente error
![error mvn clean install dbeaver](https://user-images.githubusercontent.com/52106718/168623039-b3329315-e110-42ae-a0e4-fb6e754cbbb4.png)

**Reply by miltonchirinos44** [reply]
Thanks, I solved the problem
